rust/xaynet-core/src/mask/masking.rs

```diff
@@ -39,14 +50,21 @@ pub enum UnmaskingError {
 #[derive(Debug, Error)]
 /// Errors related to the aggregation of masks and models.
 pub enum AggregationError {
-    #[error("the model to aggregate is invalid")]
-    InvalidModel,
+    // TODO rename Model -> Vector; or use MaskMany/One terminology
+    #[error("the object to aggregate is invalid")]
+    InvalidObject,
 
     #[error("too many models were aggregated for the current unmasking configuration")]
     TooManyModels,
 
-    #[error("the model to aggregate is incompatible with the current aggregated model")]
+    #[error("too many scalars were aggregated for the current unmasking configuration")]
+    TooManyScalars,
```

Comment:
Is it worth distingushing between `TooManyModels` and `TooManyScalars`? Could they be unified into an `UnmaskingError::TooManyMasks` for instance?

Reply:
I guess distinguishing the cases has the advantage when we get errors, e.g. to narrow down which mask config contributed to the error.